addon/kubeflow-operator/manifest.yaml

```diff
@@ -264,6 +256,9 @@ spec:
       overlays:
       - istio
       - application
+      parameters:
+      - name: userid-header
+        value: kubeflow-userid
```

Comment:
What is this `userid-header` for?

Reply:
That was actually the key of the solution. It identifies a HTTP header witch contains the user ID of the authenticated user.